lib/hooks/useValidateUser.ts

```diff
@@ -0,0 +1,17 @@
+import useSWR from "swr";
+
+interface UserResponse{
+  data: DbUser
+}
+const useValidateUser = (username:string)=>{
+
+  const {data, error} = useSWR<UserResponse, Error>(`user/${username}`);
```

Comment:
This is supposed to be hitting the API. Example here

https://github.com/open-sauced/insights/blob/beta/lib/hooks/useUserInsights.ts#L13